modules/db/src/main/scala/gem/dao/UserDao.scala

```diff
@@ -13,29 +13,10 @@ object UserDao {
     select("root")
 
   def select(id: String): ConnectionIO[User[Nothing]] =
-    sql"""
-      SELECT id, first, last, email, staff
-      FROM gem_user
-      WHERE id = $id
-    """.query[(String, String, String, String, Boolean)]
-       .unique.map { case (i, f, l, e, s) =>
-         User(i, f, l, e, s, Map.empty)
-       }
+    Statements.select(id).unique
```

Comment:
Yeah doobie fetches all the metadata and checks all the parameter/column mappings with the JDBC spec and complains about mismatched types and nullity. So if you try to read a `VARCHAR` into an `Int` it will fail, for instance.